packages/react/src/Delay.tsx

```diff
@@ -1,17 +1,27 @@
-import { type PropsWithChildren, useContext, useState } from 'react'
+import { type PropsWithChildren, type ReactNode, useContext, useState } from 'react'
 import { DelayDefaultPropsContext } from './contexts'
 import { useTimeout } from './hooks'
+import { assert } from './utils'
 
 export interface DelayProps extends PropsWithChildren {
   ms?: number
+  /**
+   * @experimental This is experimental feature.
+   */
+  fallback?: ReactNode
 }
 
-export const Delay = ({ ms, children }: DelayProps) => {
+export const Delay = ({ ms, children, fallback }: DelayProps) => {
   const defaultProps = useContext(DelayDefaultPropsContext)
   const delayMs = ms ?? defaultProps.ms ?? 0
-  const [isDelayed, setIsDelayed] = useState(delayMs === 0)
-  useTimeout(() => setIsDelayed(true), delayMs)
-  return <>{isDelayed ? children : null}</>
+
+  if (process.env.NODE_ENV !== 'production') {
+    assert(delayMs >= 0, 'Delay: ms prop should be greater than or equal to 0')
+  }
+
+  const [isDelaying, setIsDelaying] = useState(delayMs > 0)
+  useTimeout(() => setIsDelaying(false), delayMs)
+  return isDelaying ? (typeof fallback === 'undefined' ? defaultProps.fallback : fallback) : children
```

Comment:
I removed unnecessary jsx `<></>`

> Rather than using nested trinomials, why not just define and use the fallback once above, or even use an early return strategy to simplify the return statement!

@minsoo-web isn't it simpler than before? I don't want to make unnecessary variable name if possible.